### PR TITLE
add: card component

### PR DESCRIPTION
**Pull request by Sudhanshusoni45**
CSB link to preview : https://vc782.csb.app/Pages/Card/card.html

### CODE REVIEW

**Review by Abhijeet43 — CHANGES_REQUESTED** [review]
Hey, your text-overlay card and card with close button donot have any gap between them and also use variables for colors..Please fix it

**Review by Ayushi20-19 — APPROVED** [review]
suggestion: use variable for colors, and rem units.

**Review by Ayushi20-19 — APPROVED** [review]
suggestion: use variable for colors, and rem units.

**Review by mohammed-sas — APPROVED** [review]
pr good to merge
suggestion : try using css variables for color properties

File: Pages/Card/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5502
+}
```
Comment (VishalPatil18) [inline Pages/Card/.vscode/settings.json:3]: You can add this files path in the `.gitignore` file as it is not related to the project.

File: Pages/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+}
+
+/* lower */
+
+.lower {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 0.4rem;
+  margin-bottom: 0.4rem;
```
Comment (VishalPatil18) [inline Pages/Card/card.css:35]: You could use the shorthand of `margin: 0.4rem initial;` as it does the work in just one line.

File: Pages/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+.details p {
+  color: rgba(0, 0, 0, 0.54);
+  letter-spacing: 0.25px;
+  font-size: 14px;
+}
+
+.details {
+  margin-top: 3px;
+  padding: 0 1rem;
+  padding-bottom: 0.5rem;
```
Comment (VishalPatil18) [inline Pages/Card/card.css:112]: Here also a shorthand `padding: 0 1rem 0.5rem;` could be used as it does the work in one line. It works in this way `padding: [padding-top]   [padding-left-and-right]   [padding-bottom]`

File: Pages/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+.horizontal-card .upperSection:hover {
+  background-color: #f5f5f5;
+  cursor: pointer;
+}
+
+.horizontal-card .upperSection img {
+  width: 110px;
+  height: 110px;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
```
Comment (VishalPatil18) [inline Pages/Card/card.css:150]: A shorthand of `border-radius: 4px 4px initial initial;` could be used here.

File: Pages/Card/card.css [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+  right: -9px;
+  position: absolute;
+  font-size: 0.5rem;
+  color: white;
+  font-weight: 900;
+  height: 35px;
+  width: 35px;
+  border: none;
+  background-color: red;
+  border-radius: 50%;
```
Comment (VishalPatil18) [inline Pages/Card/card.css:215]: As there are multiple properties in this class you could sort them in alphabetical order as that helps in debugging as well as code maintainability and code readability.

File: Pages/Card/card.html [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+              <small>READ</small>
+            </button>
+            <button>
+              <small>BOOKMARK</small>
+            </button>
+          </div>
+          <div>
+            <img src="./images/love.png" alt="" />
+            <img src="./images/share.png" alt="" />
+            <img src="./images/dots.png" alt="" />
```
Comment (VishalPatil18) [inline Pages/Card/card.html:45]: Give value to the `alt` attribute as that helps in accessibility.

**Review by VishalPatil18 — APPROVED** [review]
- A few suggestions around shorthand properties and code accessibility were added.
- Also, there's no horizontal card which is a must as per the LMS requirement. You should check it out maybe in the PRD.
